src/main/java/org/example/models/JobRoleInfo.java

```diff
@@ -0,0 +1,118 @@
+package org.example.models;
+
```

Comment:
Name of the file needs to have response.

Reply:
Done